truss/tests/templates/server/test_model_wrapper.py

```diff
@@ -143,7 +143,7 @@ async def mock_predict(return_value, request):
             assert resp == expected_predict_response
 
 
-@pytest.mark.anyio
+@pytest.mark.asyncio
```

Comment:
Similar to the others, I'm not 100% sure why we have `anyio` vs `asyncio` here, since I imagine we'd want to ensure all tests use the same event loop. This was having weird interactions with the `test_graceful_shutdown`, and marking this as `asyncio` fixed it. A followup should probably deprecate `anyio`